### PR TITLE
fixed wallet adapter page

### PR DESCRIPTION
**Pull request by shayansanjideh**

### CODE REVIEW

File: docs/reference/sui-wallet-kit/wallet-kit-provider.mdx [hunk 1]
```diff
@@ -1,20 +1,12 @@
 ---
 title: Wallet Kit Provider
-sidebar_label: Wallet Kit Provider
-slug: /wallet-kit-provider
+sidebar_label: walletKitProvider
+slug: /walletKitProvider
```
Comment (bhenhsi) [inline docs/reference/sui-wallet-kit/wallet-kit-provider.mdx:4]: Are we going to use camelcase?

Comment (shayansanjideh) [reply docs/reference/sui-wallet-kit/wallet-kit-provider.mdx:4]: I used camel case only for these to be consistent with its API’s spelling.

Comment (wriches) [reply docs/reference/sui-wallet-kit/wallet-kit-provider.mdx:4]: IMO the URLs don't need to be consistent with the method names

Comment (shayansanjideh) [reply docs/reference/sui-wallet-kit/wallet-kit-provider.mdx:4]: Got it I can change it back then

**Pull request closed without merging**